snap_http/api.py

```diff
@@ -219,16 +219,68 @@ def revert_all(names: List[str]) -> SnapdResponse:
     return http.post("/snaps", {"action": "revert", "snaps": names})
 
 
-def remove(name: str) -> SnapdResponse:
+def remove(name: str,
+           purge: Optional[bool] = False,
+           terminate: Optional[bool] = False) -> SnapdResponse:
     """Uninstalls a snap identified by `name`."""
-    return http.post("/snaps/" + name, {"action": "remove"})
+    body = {
+        "action": "remove",
+        "purge": purge,
+        "terminate": terminate,
+    }
+
+    return http.post("/snaps/" + name, body)
 
 
 def remove_all(names: List[str]) -> SnapdResponse:
     """Uninstalls all snaps identified in `names`."""
     return http.post("/snaps", {"action": "remove", "snaps": names})
 
 
+def snapshots() -> SnapdResponse:
+    """Gets a list of all snapshots."""
+    return http.get("/snapshots")
+
+
+def forget_snapshot(id: str, snaps: Optional[List[str]] = None, users: Optional[List[str]] = None) -> SnapdResponse:
+    """Deletes a snapshot identified by `id`.
+
+    :param snap_id: The ID of the snapshot to delete.
+    """
+
+    body: Dict[str, Any] = {
```

Comment:
typing nit: not sure `Any` is an appropriate type here. It seems to me that the complete set of possible key types is `Union[str, List[str]]`.